src/ui/qgsgeopackageprojectstoragedialog.ui

```diff
@@ -16,12 +16,12 @@
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
-        <string>Connection</string>
+        <string>File name</string>
```

Comment:
This combo shows not just filenames, but also the names of stored geopackage connections. I don't think using "file name" is appropriate here.